styles.css

```diff
@@ -0,0 +1,739 @@
+  }
+
+  .sec1 h2 {
+    font-weight: 600;
+    font-size: 26px;
+  }
+
+  h1 {
+    font-weight: 900;
+    font-size: 48px;
+    line-height: 60px;
+    width: 90%;
+  }
+
+  .text {
+    width: 50%;
+    min-height: 100px;
+    margin-left: 240px;
+    margin-top: 10px;
+    padding-left: 25px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border: 4px solid #fff;
+  }
+
+  .place {
+    width: 50%;
+    min-height: 100px;
+    margin-left: 240px;
+    padding-left: 0;
+  }
+
+  .main-container {
+    height: 547px;
+    width: 100%;
+    margin-top: 0;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .flex-wrapper {
+    display: flex;
+    gap: 12px;
+    width: 100%;
+    min-height: 240px;
+  }
+
+  .b-grey {
+    min-height: 250px;
+    width: 50%;
+    display: flex;
+    padding-left: 13px;
+    padding-right: 13px;
+    padding-bottom: 18px;
+    padding-top: 6px;
+    flex-direction: column;
+  }
+
+  .b-grey h4 {
+    text-align: center;
+  }
+
+  .b-grey .text2 {
+    text-align: center;
+  }
+
+  .text3 {
+    display: none;
+  }
+
+  .textDesk {
+    font-family: "Lato", sans-serif;
+    font-size: 18px;
+    font-weight: 600;
+    color: #fff;
+    margin-top: 70px;
+    text-decoration: underline;
+    text-align: center;
+  }
+
+  .btnLect {
+    display: none;
+  }
+
+  .lecturer {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+  }
+
+  .lecturers {
+    width: 80%;
+    min-height: 450px;
+    margin: 36px auto;
+    display: flex;
+    flex-direction: row;
+  }
+
+  .sponsors {
+    font-family: "Lato", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 24px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .sponsors h3 {
+    margin-bottom: 15px;
+  }
+
+  .partners1 {
+    width: 100%;
+    min-height: 240px;
+    padding-top: 35px;
+    background-color: #272a31;
+    color: #8f8f91;
+    margin-bottom: 100px;
+    margin-top: -450px;
+  }
+
+  .partners1 ul {
+    display: flex;
+    list-style: none;
+    justify-content: space-between;
+    width: 70%;
+    min-height: 50px;
+    margin: 60px auto;
+    font-family: fantasy;
+    font-size: x-large;
+  }
+
+  footer {
+    font-family: "Lato", sans-serif;
+    width: 100%;
+    height: 145px;
+  }
+
+  footer img {
+    width: 250px;
+  }
+
+  .flex-cont {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0 auto;
+    width: 80%;
+    height: 98px;
+  }
+
+  footer p {
+    text-align: center;
+    font-size: 14px;
+    font-weight: 700;
+    margin-top: 0;
+  }
+
+  footer span {
+    font-weight: 900;
+    font-size: 20px;
+  }
+
+  footer .flex-f {
+    width: 100%;
+    height: 245px;
+  }
+
+  #showMore {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    width: 90%;
+    min-height: 600px;
+    margin: 30px auto;
+  }
```

Comment:
- [ ] Well done doing this section dynamically Kindly ensure that the style is working for the mobile and desktop versions, the next picture is from the desktop version, and the lectures should be in two columns for a better UI 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/235305652-0baa30d5-2935-4f05-b5bc-180f7e3f618f.png)